challenges/platform/README.md

```diff
@@ -0,0 +1,48 @@
+
+
+# Rationale
+
+These exercises have two goals.
+
+- It helps us understand what to expect from you as an engineer, how you reason about production systems and investigate faults.
+- It helps you get a feel for what it would be like to work at Teleport, as this exercise aims to simulate some of our day-as-usual and expose you to the type of work we're doing here.
+
+We believe this technique is better and is more fun compared to whiteboard/quiz interviews so common in the industry. It's not without the downsides as it could take longer than traditional interviews. [Some of the best teams use coding challenges.](https://sockpuppet.org/blog/2015/03/06/the-hiring-post/)
+
+We appreciate your time and are looking forward to hacking on this challenge together.
+
+
+# Backgrounds
+
+## Development Focus: Systems / Backend Developers
+Systems developers on the platform team leverage a strong background in software development to build, maintain, deploy core production services.
+
+### Challenge
+1. [Systems developer challenge](../systems/worker.pdf)
+
+### Levels
+See [levels.pdf](../../levels.pdf)
+
+
+## Operations Focus: Devops / DevSecOps / SREs / Production engineers / etc
+Team members with experience in various operations focused disciplines are focussed on the day to day delivery of platform components, such as build/testing automation of infrastructure as code, deploying and configuring our metrics, alarming and logging systems, automating build pipeliness, and other day to day activities to enable our cloud infrastructure.
+
+
+### Challenge
+We use two interviews for team members with operational backgrounds, the first is a lightweight scripting / programming challenge, and the second is a troubleshooting production test. See the following links for outlines of each challenge.
+1. [Devops development challenge](devops.md)
+2. [Troubleshooting challenge](troubleshooting.md)
+
+### Levels
+
+
+| Level  | Oncall  | Operations  | Security  | Communications / Writing  | Networking  | Systems Engineering  | OS Understanding  | Tooling |
+|---|---|---|---|---|---|---|---|---|
+| 1 | Participates in on-call and able to execute run books | Documents and Executes maintenance activities in production | Able to follow security guidelines and best practices in a production environment  | Contributor to internal documentation such as updating run books | Able to troubleshoot introductory networking issues, network policies, routing, etc. | Focus on and contribute to specific area of production system (such as the deployment of our monitoring stack)  | Working understanding of core OS concepts, filesystem, networking, cgroups, namespaces, security mechanisms, etc.  | Work with team members on deployment and maintenance of internal tooling |
```

Comment:
I would zoom in here on skills and technologies that are relevant to the team. This leveling does not have to be universally valid, just valid for the cloud team. For example, as a L1 engineer, reading this level, I should ideally get a good answer, what technology and skill should I understand and practice to get to L2 and how do I demonstrate that? 

For example, I would elaborate on:

Solid understanding of advanced networking, routing, load balancing concepts and scaling production systems

We probably mean something quite specific here, like container networking, ability to set-up VPNs and tunnels, traffic routing, etc. Someone reading this would then understand what's required and how to get there.

Reply:
I rotated the table so hopefully it formats better and did a bit of tuning to the language to include more examples, but probably requires more work still.